pgjdbc/src/main/java/org/postgresql/jdbcurlresolver/PgPassParser.java

```diff
@@ -67,21 +66,17 @@ private PgPassParser(String hostname, String port, String database, String user)
     if (user == null || user.isEmpty()) {
       return null;
     }
-    PgPassParser pgPassParser = new PgPassParser(hostname, port, database, user);
+    PgPassParser pgPassParser = new PgPassParser(fileName, hostname, port, database, user);
     return pgPassParser.findPassword();
   }
 
   private @Nullable String findPassword() {
-    String resourceName = findPgPasswordResourceName();
-    if (resourceName == null) {
-      return null;
-    }
-    //
     String result = null;
```

Comment:
If we are looking to provide psql compatibility we should probably check the permissions of the file to ensure they are 0600 on *nix systems.

Reply:
Good observation. Permission check is added: https://github.com/pgjdbc/pgjdbc/compare/581edc3..736f2cb